File: xapp/geany/files/patch-x-data-templates-files-file.pl
--- ./data/templates/files/file.pl.orig	2022-12-12 21:37:08.520578000 +0200
+++ ./data/templates/files/file.pl	2022-12-17 00:46:42.275540000 +0200
@@ -0,0 +1,45 @@
+#!/usr/bin/env perl
+
+{fileheader}
+
+package Hello;
+
+use strict;
+use warnings;
+
+use Exporter qw(import);
+our @EXPORT_OK = qw(hello);
+
+sub hello {
+    print "hello\n";
+}
+
+
+sub new {
+    my $class = shift;
+    my $self = {
+    };
+    bless $self, $class;
+    return $self;
+}
+
+sub print {
+    my $class = shift;
+    print "hello\n";
+}
+
+1;
+
+package main;
+
+use lib "libs";
+
+use strict;
+use warnings;
+use Hello::More qw(hello);
+use Hello;
+
+hello();
+
+my $hello = Hello->new;
+$hello->print;

File: xapp/geany/files/patch-x-data-templates-files-file.go
--- ./data/templates/files/file.go.orig	2019-12-30 08:34:58.340214000 +0200
+++ ./data/templates/files/file.go	2019-12-30 08:34:58.340297000 +0200
@@ -0,0 +1,11 @@
+{fileheader}
+package main
+
+import (
+    "fmt"
+)
+
+func run() error {
+    var err error
+    fmt.Printf("hello!\n")
+    return err
+}
+
+func main() {
+    err := run()
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+    }
+}
